test/test_multizmq_io.py

```diff
@@ -9,14 +9,15 @@
 from larpix.io.multizmq_io import MultiZMQ_IO
 from larpix.format.message_format import dataserver_message_decode, dataserver_message_encode
 
-def test_generate_chip_key():
+@pytest.mark.skip
```

Comment:
I recommend not skipping these tests. They should all be able to function even with instances of MultiZMQ_IO initialized.